Vagrantfile

```diff
@@ -55,7 +55,7 @@ end
 
 # TODO(ksweeney): RAM requirements are not empirical and can probably be significantly lowered.
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
-  config.hostmanager.enabled = true
+  config.hostmanager.enabled = false
```

Comment:
Also, related is that it seems a bit painful to make it harder to use the standard vagrant tooling. Maybe we have to in order to have a workable solution across both Virtualbox and AWS, but I missed the `vagrant hostmanager` step with muckrake today and it was only obvious to me what went wrong because I knew about these patches and realized I shouldn't have used `vagrant up` directly.

Another idea might be to disable hostmanager here but use it's [provisioner](https://github.com/smdahlen/vagrant-hostmanager#provisioner) to make sure bringing up non-worker clusters works properly and maybe forcing it to run when the last node is brought up (so we don't require a separate step). I haven't looked into the provisioner carefully though, so I'm not sure of its exact behavior.